pkgs/applications/version-management/vcsh/default.nix

```
@@ -1,26 +1,42 @@
-{ lib, stdenv, fetchFromGitHub, which, git, ronn, perlPackages }:
+{ lib, stdenv
+, fetchurl
+, makeWrapper
+, pkg-config
+, autoconf
+, automake
+, git
+, perlPackages
+}:
 
 stdenv.mkDerivation {
```

Maybe use `mkDerivation rec` to re-use `version` in `src`.

```suggestion
stdenv.mkDerivation rec {
```